shared/src/main/java/net/blay09/mods/waystones/item/CrumblingAttunedShardItem.java

```diff
@@ -0,0 +1,151 @@
+package net.blay09.mods.waystones.item;
+
+import net.blay09.mods.balm.api.Balm;
+import net.blay09.mods.waystones.api.IAttunementItem;
+import net.blay09.mods.waystones.api.IFOVOnUse;
+import net.blay09.mods.waystones.api.IResetUseOnDamage;
+import net.blay09.mods.waystones.api.IWaystone;
+import net.blay09.mods.waystones.block.WarpPlateBlock;
+import net.blay09.mods.waystones.compat.Compat;
+import net.blay09.mods.waystones.config.WaystonesConfig;
+import net.blay09.mods.waystones.core.PlayerWaystoneManager;
+import net.blay09.mods.waystones.core.WarpMode;
+import net.blay09.mods.waystones.core.WaystoneProxy;
+import net.blay09.mods.waystones.menu.WarpPlateContainer;
+import net.minecraft.ChatFormatting;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.NbtUtils;
+import net.minecraft.nbt.Tag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.InteractionResultHolder;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+import java.util.Objects;
+
+public class CrumblingAttunedShardItem extends ScrollItemBase implements IAttunementItem, IFOVOnUse, IResetUseOnDamage {
```

Comment:
These should not be usable like scrolls imo - we already have Bound Scrolls and having those be bindable to a warp plate (right click or by placing inside) would be much more consistent design-wise

Reply:
Maybe right-clicking a warp stone with a crumbling shard could consume it to "charge" the warp plate for one teleport to that destination? It could go along with some special particles forming at its base or something. Though I'd recommend splitting that out into a separate PR to keep things focused to one feature at a time